xdsl/interactive/app.py

```diff
@@ -165,14 +170,15 @@ class InputApp(App[None]):
     (i.e. is the Output TextArea).
     """
     pass_pipeline = reactive(tuple[tuple[type[ModulePass], PipelinePassSpec], ...])
-    """Reactive variable that saves the list of selected passes."""
+    """Reactive variable that saves the selected passes."""
 
     condense_mode = reactive(False, always_update=True)
     """Reactive boolean."""
-    available_pass_list = reactive(tuple[type[ModulePass], ...])
+    available_pass_list = reactive(
+        tuple[tuple[str, type[ModulePass], PipelinePassSpec | None], ...]
```

Comment:
Can you please factor out this tuple?
```suggestion
        tuple[AvailablePass, ...]
```

``` python

class AvailablePass(NamedTuple):
    display_name: str
    module_pass: type[ModulePass]
    pass_spec: PipelinePassSpec | None
```